src/main/java/org/atlasapi/persistence/content/mongo/FullMongoScheduleRepopulator.java

```
@@ -35,11 +36,11 @@ public FullMongoScheduleRepopulator(ContentLister contentLister, ChannelResolver
         this.contentLister = contentLister;
         this.scheduleStore = scheduleStore;
         this.publishers = ImmutableList.copyOf(publishers);
-        this.scheduleEntryBuilder = new ScheduleEntryBuilder(channelResolver, maxBroadcastAge);
+        this.scheduleEntryBuilder = new ScheduleEntryBuilder(channelResolver);
```

This is wrong. This constructor provides a `maxBroadcastAge`. Silently ignoring it will confuse someone using this constructor.

You need to maintain the existing behaviour. Do you need to change this class? You only care about schedule changes during the normal write process. If I'm not mistaken this is a repopulator which you are not using therefore you shouldn't need to change it at all